.bazelrc

```diff
@@ -375,7 +375,7 @@ build:release_windows --config=release
 # configuration, but for clang-cl this is equivalent to only -O2 and a few other things. -O3 is
 # not exposed directly in the clang-cl driver, but we can access regular clang
 # flags using the /clang prefix anyway. https://clang.llvm.org/docs/UsersManual.html#the-clang-option
-build:release_windows --copt="/clang:-O3"
+build:release_windows --copt="/clang:-O2"
```

Comment:
As discussed using -O2 appears to be causing issues too, but we should limit this to only the offending file: `--per_file_copt=capnp/rpc.c++@-O1`